packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/favorites/GetFavoritesTests.java

```diff
@@ -555,14 +561,28 @@ public void checkResponseSchemaForGetFavorites() throws Exception
 
     @Test(groups = { TestGroup.REST_API, TestGroup.FAVORITES, TestGroup.REGRESSION })
     @TestRail(section = { TestGroup.REST_API, TestGroup.FAVORITES }, executionType = ExecutionType.REGRESSION, description = "Verify if get favorites response returns allowableOperations object when requested")
-    public void checkResponseForGetFavoritesWithAllowableOperations()
-    {
-        RestPersonFavoritesModelsCollection adminFavorites =
+    public void checkResponsesForGetFavoritesWithAllowableOperationsAndSearchRequestWithIsFavorite() throws InterruptedException {
```

Comment:
Trailing brace formatting. We should get around to investigating pre-commit formatters.

Reply:
Should be okay now